### PR TITLE
Fix metaclass usage for six 1.11.0.

### LINKED ISSUE #24: six 1.11.0 causes TypeError: Error when calling the metaclass bases
This is the same error seen in apitools here: https://github.com/google/apitools/issues/175

### PR DESCRIPTION
**Pull request by inklesspen**
Fixes #24.

### CODE REVIEW

File: protorpc/messages.py [hunk 1]
```diff
@@ -1152,7 +1152,6 @@ class Field(six.with_metaclass(_FieldMeta, object)):
   #
   # We add an alias here for backwards compatibility; note that in
   # python3, this attribute will silently be ignored.
-  __metaclass__ = _FieldMeta
```
Comment (kryzthov) [inline protorpc/messages.py]: Maybe drop the comment/TODO along the way?

Comment (inklesspen) [reply protorpc/messages.py]: Done, thanks.

**Comment by kryzthov** [comment]
LGTM

**Comment by craigcitro** [comment]
Thanks!

**Pull request merged**